smz

```diff
@@ -55,6 +56,15 @@ sub_backlinks(){
     search_in_files "$search_pattern" "$SMZPATH" --files-with-matches | grep -v "$1"
 }
 
+sub_orphans(){
+    for zettel in "$SMZPATH"/*
+    do
+      if [ -z "$(smz b "$zettel")" ]; then
```

Comment:
There's probably a way to do this and avoid the subshell, but this seems to work just fine.